### ISSUE TITLE
`<Pds::TabNav>` causes accessibility violations when used as documented

### ISSUE THREAD

**Issue by jgwhite** [hashicorp/structure#100]
Using `<Pds::TabNav>` as documented causes aXe violations.

For example, in the Waypoint codebase we use `<Pds::TabNav>` [in the app.hbs template](https://github.com/hashicorp/waypoint/blob/v0.4.1/ui/app/templates/workspace/projects/project/app.hbs#L53-L79) and we see the following violations from aXe (via ember-a11y-testing):

```
Promise rejected during "visiting builds page":

  The page should have no accessibility violations. Violations:

    [minor]: ARIA role should be appropriate for the element 
      Violated 1 time. Offending nodes are: 
        <nav class="pds-tabNav" role="tablist" data-test-tab-nav="">
      https://dequeuniversity.com/rules/axe/4.2/aria-allowed-role?application=axeAPI

    [critical]: Certain ARIA roles must contain particular children 
      Violated 1 time. Offending nodes are: 
        <nav class="pds-tabNav" role="tablist" data-test-tab-nav="">
      https://dequeuniversity.com/rules/axe/4.2/aria-required-children?application=axeAPI
```

**Comment by didoo** [comment]
Outdated. Closing now.
